public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,178 @@
+# dvc.api.open()
+
+[Context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
```

Comment:
too complicated, tech jargon ... even if want to mention that it can be used with `with`, we need to do it a second sentence

Reply:
Has been updated.